scripts/ci/check_compliance.py

```diff
@@ -1084,6 +1087,48 @@ def run(self):
             except MaintainersError as ex:
                 self.failure(f"Error parsing {file}: {ex}")
 
+class ModulesMaintainers(ComplianceTest):
+    """
+    Check that all modules have a MAINTAINERS entry.
+    """
+    name = "ModulesMaintainers"
+    doc = "Check that all modules have a MAINTAINERS entry."
+    path_hint = "<git-top>"
+
+    def run(self):
+        MAINTAINERS_FILES = ["MAINTAINERS.yml", "MAINTAINERS.yaml"]
+
+        manifest = Manifest.from_file()
+
+        try:
+            manifest_file = Path(manifest.path).relative_to(ZEPHYR_BASE)
```

Comment:
Just a note that `manifest.path` is deprecated and undocumented in https://docs.zephyrproject.org/latest/develop/west/west-apis.html#west.manifest.Manifest; `manifest.abspath` is preferred these days.

Reply:
Cool, ended up dropping it anyway.